pom.xml

```diff
@@ -60,8 +60,8 @@
         <junit5.version>5.7.0</junit5.version>
         <hamcrest-library.version>2.2</hamcrest-library.version>
         <hdr-histogram-version>2.1.12</hdr-histogram-version>
-        <kotlin.version>1.3.72</kotlin.version>
-        <kotlinx-coroutines.version>1.3.8</kotlinx-coroutines.version>
+        <kotlin.version>1.4.21</kotlin.version>
```

Comment:
Can you split the commits into dependency upgrades and formatting fixes? Feel free to commit and push the dependency upgrades directly to `main`.

Reply:
Sure, done